.yarnrc.yml

```diff
@@ -0,0 +1,9 @@
+nodeLinker: node-modules
+npmRegistryServer: https://registry.npmjs.org
+plugins:
```

Comment:
Why these plugins?

Reply:
These are "core plugins" that people expect to be present from Yarn Classic. I'm including them on all Yarn Berry PRs for convenience. Yarn v4 (which was recently released) includes core plugins by default again.